relay-server/src/actors/processor.rs

```diff
@@ -2604,17 +2609,9 @@ impl EnvelopeProcessorService {
                 received_at: Some(state.managed_envelope.received_at()),
                 max_secs_in_past: Some(self.inner.config.max_secs_in_past()),
                 max_secs_in_future: Some(self.inner.config.max_secs_in_future()),
-                transaction_range: Some(
-                    self.inner
-                        .config
-                        .aggregator_config_for(MetricNamespace::Transactions)
-                        .timestamp_range(),
-                ),
+                transaction_range: Some(agg.timestamp_range()),
                 max_name_and_unit_len: Some(
-                    self.inner
-                        .config
-                        .aggregator_config()
-                        .max_name_length
+                    agg.max_name_length
```

Comment:
Drive-by fix: this was reading from the wrong aggregator config (without consequences because we rely on defaults anyway).